apps/admin/frontend/src/utils/manual_tallies.ts

```diff
@@ -246,8 +244,7 @@ export function convertTalliesByPrecinctToFullManualTally(
 }
 
 export function getEmptyContestTallies(
-  election: Election,
-  allAdjudicatedValues?: Map<ContestId, string[]>
```

Comment:
Because of the change in contract (we don't need to include write-ins in contestTallies if they're 0, and we have a different source of truth for who the write-in candidates are), we don't need to include these names from adjudicated values.